.github/workflows/build.yml

```diff
@@ -17,8 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        # only use one version for the lint step
-        python-version: ["3.11"]
+        python-version: ["3.11", "3.12"]
```

Comment:
Adding coverage for python 3.12 seems unrelated to transitioning to `pyproject.toml`.

Reply:
Indeed - reverted the change